src/openforms/js/components/admin/forms/VariableSelection.js

```diff
@@ -30,20 +30,16 @@ const VariableSelection = ({
       const label = formDefinitionsNames[variable.formDefinition]
         ? `${formDefinitionsNames[variable.formDefinition]}: ${variable.name} (${variable.key})`
         : `${variable.name} (${variable.key})`;
-      return [variable.key, label];
+      return {value: variable.key, label};
```

Comment:
This doesn't appear to implement the `label` JSX with nicer markup/styling yet, right? Something like:

```jsx
<div className="variable-option">
    <span className="variable-option__name">{variable.name}<span>
    <span className="variable-option__key">({variable.key}}<span>
    {formDefinitionsNames[variable.formDefinition] && <span className="variable-option__context">{formDefinitionsNames[variable.formDefinition]}<span>}
</div>
```

Reply:
I've made it into something fancy ✨